typescript/db/src/resolvers/workspace.ts

```diff
@@ -136,7 +132,7 @@ const deleteWorkspace = async (
   const db = await getPrismaClient();
   const data = await db.workspace.findUnique({ where: { id } });
   foundWorkspace(data, args.where);
-  return addTypename(data);
+  return addTypenameToData(data, "Workspace");
```

Comment:
Why didn't you kept the `addTypename` function name which was fine IMO?

Reply:
because it felt easier to name `addTypenameToDataArray` along with `addTypenameToData`, with `addTypename` I wasn't sure how to clearly name the array version, but I'm open to suggestions.

Reply:
Why not `addTypenames`?

Reply:
Too easy to confuse and miss the s when re-reading the code, and typing was strong enough to make it fail.
However with your subjected changes on typing, it does properly prevent from misusing the functions, so I will accept your suggestion to revert to `addTypename(s)`